acl/index.js

```diff
@@ -12,49 +12,69 @@ var async = require('async');
 var wsModels = require('loopback-workspace').models;
 var ModelAccessControl = wsModels.ModelAccessControl;
 
-var actions = require('../lib/actions');
 var helpers = require('../lib/helpers');
 var helpText = require('../lib/help');
+var ActionsMixin = require('../lib/actions');
+var debug = require('debug')('loopback:generator:acl');
+
+module.exports = class ACLGenerator extends ActionsMixin(yeoman) {
+  constructor(args, opts) {
+    super(args, opts);
+  }
 
-module.exports = yeoman.Base.extend({
   // NOTE(bajtos)
   // This generator does not track file changes via yeoman,
   // as loopback-workspace is editing (modifying) files when
   // saving project changes.
 
-  help: function() {
+  help() {
     return helpText.customHelp(this, 'loopback_acl_usage.txt');
-  },
-
-  loadProject: actions.loadProject,
-
-  loadModels: actions.loadModels,
-
-  loadAccessTypeValues: function() {
+  }
+
+  loadProject() {
+    debug('loading project...');
```

Comment:
@dhmlau I think the progress info like ^ is fine, since it doesn't contain any information.
I will double check all the places that prints variables....I don't see any sensitive info in the generator, while some probably prints long nested object. Will remove them.

Reply:
ok.  I have this question because it may not seem useful in the future for debugging purpose.  If you think it's needed, i'm ok to keep it. thanks.